.gitignore

```diff
@@ -1,5 +1,5 @@
 db.sqlite3
 .vscode/
 FRSITE_VENV/
-
+**/__pycache__/*
```

Comment:
I noticed that this file creeps up a lot when running the project, ignoring it helps with sanity